### PR TITLE
Added bash script for iDiff tests

### PR DESCRIPTION
**Pull request by cftorres**
@aaron-prindle @abbytiz @nkubala

### CODE REVIEW

File: iDiff/.iDiffTests.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+go run iDiff/main.go iDiff 0cb40641836c e7d168d7db45 dir -j > iDiff/tests/runs/busybox_diff_actual.json
```
Comment (abbytiz) [inline iDiff/.iDiffTests.sh]: To clarify, once the URL PR is in this will be changed to image URL's correct?

Comment (cftorres) [inline iDiff/.iDiffTests.sh]: yes!

**Comment by aaron-prindle** [comment]
It seems this requires the URL PR to go in.  Get that merged first, then rewrite it using URLs pointing to public images.  Then the test will work like normal with Travis and we can use that to validate correctness.

**Comment by aaron-prindle** [comment]
Once this is resubmitted and the tests past we can merge.

**Pull request merged**